ui/src/tasks/components/TaskRunsList.tsx

```diff
@@ -104,9 +104,18 @@ export default class TaskRunsList extends PureComponent<Props, State> {
 
   public listRuns = (runs: Run[]): JSX.Element => {
     const {taskID} = this.props
+
+    let recentRuns = []
```

Comment:
You could just use `const recentRuns = runs.slice(19)` here.

Reply:
I tried that but it messes up the sorting. Doesn't show the latest test runs first.